src/state/assets/userAssets.ts

```diff
@@ -63,6 +63,7 @@ function serializeUserAssetsState(state: Partial<UserAssetsState>, version?: num
       chainBalances: state.chainBalances ? Array.from(state.chainBalances.entries()) : [],
       idsByChain: state.idsByChain ? Array.from(state.idsByChain.entries()) : [],
       userAssets: state.userAssets ? Array.from(state.userAssets.entries()) : [],
+      searchCache: undefined,
```

Comment:
ignore serialization of `searchCache` which as a `Map` object would always only be serialized to `{}` regardless of what was inside of it

Reply:
I think instead of doing this, you could just exclude it from the persistence process by removing it from the partializer:

```
      partialize: state => ({
        chainBalances: state.chainBalances,
        idsByChain: state.idsByChain,
        userAssets: state.userAssets,
      }),
```

See how we do it in the browser store [here](https://github.com/rainbow-me/rainbow/blob/3d5a7704133816e701c3a7a7478ccf179c89a1b9/src/state/browser/browserStore.ts#L213)